common/djangoapps/student/views.py

```diff
@@ -1541,7 +1541,7 @@ def user_signup_handler(sender, **kwargs):  # pylint: disable=unused-argument
             log.info(u'user {} originated from a white labeled "Microsite"'.format(kwargs['instance'].id))
 
 
-def _do_create_account(form, custom_form=None):
+def _do_create_account(form, custom_form=None, site=None):
```

Comment:
The commands do not need to be modified. They can continue to call this function without the `site` keyword arg.